rules/CodeQuality/Enum/ToManyMappings.php

```diff
@@ -0,0 +1,13 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Rector\Doctrine\CodeQuality\Enum;
+
+class ToManyMappings
+{
+    /**
+     * @var class-string[]
+     */
+    public const TO_MANY_CLASSES = ['Doctrine\\ORM\\Mapping\\OneToMany', 'Doctrine\\ORM\\Mapping\\ManyToMany'];
```

Comment:
Extracted relation mapping classes to use them in multiple places not to duplicate code.

Initially I created two consts: ONE_TO_MANY_CLASS and MANY_TO_MANY_CLASS, but phpstan complained that they are not used anywhere despite they were used in TO_MANY_CLASSES